src/lib.rs

```diff
@@ -1349,7 +1386,21 @@ impl Octocrab {
 
         let token_object =
             InstallationToken::from_response(crate::map_github_error(response).await?).await?;
-        token.set(token_object.token.clone());
+
+        let expiration = token_object
+            .expires_at
+            .map(|time| {
+                DateTime::<Utc>::from_str(&time).map_err(|e| error::Error::Other {
```

Comment:
Github doesn't document the timezone but I assumed it was UTC. Also checked a couple responses manually and verified that it was UTC.